docker-compose-ppm-nsv.yml

```diff
@@ -4,35 +4,34 @@
 
 version: '3'
 services:
-  zookeeper:
-    image: wurstmeister/zookeeper
-    ports:
-      - "2181:2181"
-
   kafka:
-    image: wurstmeister/kafka
+    image: bitnami/kafka:latest
+    hostname: kafka
     ports:
       - "9092:9092"
+      - "9093:9093"
```

Comment:
What do the additional ports provide?

Reply:
These additional ports allow communication between the KRaft controller and broker

Reply:
Ah ok, does the KRaft controller and broker communicate external to the docker-compose network? This should be exposing those ports on the local machine, right?